src/llm/__init__.py

```diff
@@ -1,129 +1,27 @@
-
-
-def get_most_relevant_contents_from_message(message, top=5):
-    message_embedding = generate_embedding(message)
-    possible_contents = session.scalars(
-        select(CompanyContent)
-        .filter(CompanyContent.embedding.l2_distance(message_embedding) < 1)
-        .order_by(CompanyContent.embedding.l2_distance(message_embedding).asc())
-        .limit(top)
-    ).all()
-    return possible_contents
-
-
-async def process_user_intent(session_id, message):
-    """
-    Process user intent using memory and embeddings
-    """
-    # top 2 most relevant contents
-    relevant_contents = get_most_relevant_contents_from_message(message, top=2)
-
-    if len(relevant_contents) == 0:
-        prompt_templating = [
-            SystemMessagePromptTemplate.from_template(FALLBACK_PROMPT),
-            HumanMessagePromptTemplate.from_template("{user_message}"),
-        ]
-        relevant_contents = []
-    else:
-        suggested_content = "\n\n".join(
-            [f"{c.question}\n{c.content}\n\n" for c in relevant_contents]
-        )
-
-        prompt_templating = [
-            SystemMessagePromptTemplate.from_template(PROMPT.get("header")),
-            MessagesPlaceholder(variable_name="chat_history"),
-        ]
-
-    # append prompt content subcategory
-    if PROMPT.get("subcategory"):
-        subprompt_subcategory = PROMPT.get("subcategory")
-        for c in relevant_contents:
-            if subprompt_subcategory.get(c.subcategory):
-                subprompt = subprompt_subcategory.get(c.subcategory)
-                prompt_templating.append(
-                    SystemMessagePromptTemplate.from_template(
-                        f"{subprompt.get('header')}\n\n"
-                    )
-                )
-
-    # append prompt content suggestions
-    if len(relevant_contents) > 0:
-        prompt_templating.append(
-            SystemMessagePromptTemplate.from_template(
-                f"{PROMPT.get('suggested')}\n\n{suggested_content}"
-            )
-        )
-
-    prompt_templating.append(HumanMessagePromptTemplate.from_template("{user_message}"))
-
-    prompt = ChatPromptTemplate(messages=prompt_templating)
-
-    conversation_options = {
-        "llm": CHAT_LLM,
-        "prompt": prompt,
-        "verbose": VERBOSE_LLM
-    }
-    if PROMPT.get("memory", True):
-        chat_memory = generate_memory_instance(session_id)
-        buffer_memory = {
-            "chat_memory": chat_memory,
-            "memory_key": "chat_history",
-            "return_messages": True,
-            "k": PROMPT.get("memory_size", 5),
-        }
-        memory = ConversationBufferWindowMemory(**buffer_memory)
-        conversation_options["memory"] = memory
-    conversation = LLMChain(**conversation_options)
-    ai_message = conversation(
-        {
-            "user_message": message,
-        }
-    )
-
-    # categorize conversation history in background
-    asyncio.create_task(categorize_conversation_history(chat_memory))
```

Comment:
Is this no longer necessary?

Reply:
fat finger o'er here, sorry! getting it back